src/commands/setheadercolumncommand.js

```diff
@@ -61,21 +61,27 @@ export default class SetHeaderColumnCommand extends Command {
 	 * When the selection is already in a header column, it will set `headingColumns` so the heading section will end before that column.
 	 *
 	 * @fires execute
+	 * @params {Boolean} [forceValue] If set, the command will set (`true`) or unset (`false`) header columns according to `forceValue`
+	 * parameter instead of the current model state.
 	 */
-	execute() {
+	execute( forceValue = null ) {
```

Comment:
We already used that name in block quote and attribute command so I just used it here. Basically, yes, if you use the command with `forceValue` set, the command will set that value. It boils down to whether anything should happen at all (in case of boolean command value) but it is needed to create suggestions that aren't dependent on the current content state but the state that was at the time of creating a suggestion.

I mean, I hoped that docs are clear enough :)

```
	 * @params {Boolean} [forceValue] If set, the command will set (`true`) or unset (`false`) header columns according to `forceValue`
	 * parameter instead of the current model state.
```